tests/files/packet_centos8-crio.yml

```diff
@@ -1,15 +1,15 @@
 ---
 # Instance settings
-cloud_image: centos-7
+cloud_image: centos-8
 mode: default
 
 # Kubespray settings
 deploy_netchecker: true
 dns_min_replicas: 1
 container_manager: crio
 
-download_localhost: false
-download_run_once: true
-
 # CRI-O requirements
 etcd_deployment_type: host
+
+# required / not autodetected for now
+calico_iptables_backend: "NFT"
```

Comment:
Auto is supported now (but not the default)

Reply:
Thanks, updated for both jobs using this